libs/logging/bacon.toml

```diff
@@ -0,0 +1,78 @@
+# This is a configuration file for the bacon tool
+# More info at https://github.com/Canop/bacon
```

Comment:
Not familiar with bacon but it looks like an editor config tool? Should this be added to `.gitignore`?

Reply:
It's actually an auto test/clippy/check runner that I recommend installing. I'd like to have a way to share these settings throughout the workspace, but I just copied this from `ballot-interpreter` for now.